pkg/apis/v2beta2/receiver_webhook.go

```diff
@@ -165,6 +169,48 @@ func (r *Receiver) validateReceiver() error {
 		}
 	}
 
+	if r.Spec.Pushover != nil {
+		// validate User Profile
+		if len(r.Spec.Pushover.Profiles) == 0 {
+			allErrs = append(allErrs, field.Required(field.NewPath("spec").Child("pushover").Child("profiles"),
+				"must be specified"))
+		} else {
+			// requirements
+			tokenRegex := regexp.MustCompile(`^[A-Za-z0-9]{30}$`)
```

Comment:
It is no need to perform regular check here. You can use this annotation 
`// +kubebuilder:validation:Pattern` to achieve regular verification.
See [here](https://book.kubebuilder.io/reference/markers/crd-validation.html).

Reply:
Resolved: https://github.com/txfs19260817/notification-manager/blob/b6387586c55f6e2581fe431be82374a9b994be37/pkg/apis/v2beta2/receiver_types.go#L207